src/main/webapp/index.html

```diff
@@ -21,15 +21,18 @@ <h3>Quote of the Day</h3>
 
       <h3>Journal</h3>
       <hr/>
-      <form action="/my-data-url" method="POST">
+      <form id="form" onSubmit="return getLyrics()" action="/my-data-url" method="POST">
         <p>Enter your journal entry for the day:</p>
         <textarea name="text" maxlength="300" rows="4" cols="50" placeholder="How was your day?" required></textarea>
         <p>Enter your mood on a scale from 1-10:</p>
         <input type="number" name="mood" min="1" max="10" value="1">
         <p>Enter a song name</p>
-        <textarea name="song" maxlength="50" rows="1" cols="50" placeholder="Song name" required></textarea>
+        <textarea id="song" name="song" maxlength="50" rows="1" cols="50" placeholder="Song name" required></textarea>
         <p>Enter the artist name</p>
-        <textarea name="artist" maxlength="100" rows="1" cols="50" placeholder="Artist name" required></textarea>
+        <textarea id="artist" name="artist" maxlength="100" rows="1" cols="50" placeholder="Artist name" required></textarea>
+        <p>Enter your mood on a scale from 1-10:</p>
+        <input type="number" name="mood" min="1" max="10" value="1">
```

Comment:
There is code duplication here for the p and input elements that correspond to the mood as seen above, could you delete this please?

Reply:
I think I messed up when I did merged in master